### DISCUSSION TITLE
Proposal: Make the c# language ready for lop

### DISCUSSION THREAD

**Discussion by furesoft** [dotnet/csharplang#1966] [Language Ideas]
hi, i found this project:https://github.com/xs-admin/Excess
its really cool. so why not make c# extensible with custom syntax?

**Comment by HaloFour** [comment]
If any syntax could be legal syntax that would severely impede the team from being able to further evolve the language without causing breaking changes.

**Comment by Joe4evr** [comment]
See also the discussion at #83.
